code/GrubsGame.cs

```diff
@@ -38,6 +40,9 @@ public GrubsGame()
 			Game.SetRandomSeed( (int)(DateTime.Now - DateTime.UnixEpoch).TotalSeconds );
 			PopulatePlayerColors();
 		}
+
+		if ( GrubsConfig.SoundboardEnabled )
+			LoadSoundboard();

```

Comment:
What happens if `SoundboardEnabled` changes mid-game, are the sounds not loaded?